tests/storage/migration.go

```diff
@@ -388,6 +390,43 @@ var _ = SIGDescribe("[Serial]Volumes update with migration", Serial, func() {
 			Expect(len(migList.Items)).Should(BeNumerically(">", 1))
 			Expect(len(migList.Items)).Should(BeNumerically("<", 56))
 		})
+
+		It("should set the restart condition since the second volume is RWO and not part of the migration", func() {
+			const volName = "vol0"
+			dv1 := createDV()
+			dv2 := createBlankDV()
+			destDV := createBlankDV()
+			vmi := libvmi.New(
+				libvmi.WithNamespace(ns),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(virtv1.DefaultPodNetwork()),
+				libvmi.WithResourceMemory("128Mi"),
+				libvmi.WithDataVolume(volName, dv1.Name),
+				libvmi.WithDataVolume("vol1", dv2.Name),
+				libvmi.WithCloudInitNoCloud(libvmici.WithNoCloudEncodedUserData("#!/bin/bash\necho hello\n")),
```

Comment:
Can `WithDummyCloudForFastBoot` be used here?

Reply:
Done, thanks!